docs/src/tutorial.rst

```diff
@@ -274,10 +274,12 @@ values:
                     // scaling factor for the axis, between 1 and 100
                     "factor": 55,
                     // mode to scale the markers with respect to the properties
-                    "scale": "constant" | "inverse" | "linear" | "log" | "sqrt",
+                    "mode": "constant" | "inverse" | "linear" | "log" | "sqrt",
                     // name of the property to use for the markers size, this
                     // must be one of the key from the root `properties` table.
                     "property": "<name>",
+                    // whether to reverse the marker scaling
+                    "reversed": false | true,
```

Comment:
the property being called `reverse`, the setting should also be called `reverse`